hack/ci/run-chart-test.sh

```diff
@@ -44,8 +44,20 @@ chart::lint() {
 }
 
 chart::install_and_test() {
+    pushd "${LOCAL_REPO_ROOT_DIR}"
```

Comment:
Do we need these `pushd` and `popd` commands?

Reply:
yes, because we are using the `make` commands